Installation/changes.html

```diff
@@ -235,6 +235,9 @@ <h2 id="release4.11">Release 4.11 </h2>
       as seam edges.
     </li>
   </ul>
+     <li>Add the functions <code>read_off()</code> and <code>write_off()</code>. 
+    </li>
+</ul>
```

Comment:
That patch is not well-formed: it adds a `</ul>` for an `<ul>` that does not exist!